R/nested_model_fit.R

```diff
@@ -0,0 +1,134 @@
+#' Axing a nested_model_fit.
+#'
+#' nested_model_fit objects are created from the \pkg{nestedmodels}
+#' package, which allows parsnip models to be fitted on nested data. Axing a
+#' nested_model_fit object involves axing all the inner model_fit objects.
+#'
+#' @inheritParams butcher
+#'
+#' @seealso [axe-model_fit]
+#'
+#' @return Axed nested_model_fit object.
+#'
+#' @examplesIf rlang::is_installed(c("parsnip", "nestedmodels"))
+#'
+#' library(nestedmodels)
+#' library(parsnip)
+#'
+#' model <- linear_reg() %>%
+#'   set_engine("lm") %>%
+#'   nested()
+#'
+#' nested_data <- tidyr::nest(example_nested_data, data = -id)
+#'
+#' fit <- fit(model, z ~ x + y + a + b, nested_data)
+#'
+#' # Reduce the model size
+#' butcher(fit)
+#'
+#' @name axe-nested_model_fit
+NULL
+
+#' Remove the call.
+#'
+#' @rdname axe-nested_model_fit
+#' @export
+axe_call.nested_model_fit <- function(x, verbose = FALSE, ...) {
+  old <- x
+  x$fit$.model_fit <- purrr::map(
+    x$fit$.model_fit,
+    axe_call,
+    verbose = FALSE,
```

Comment:
```suggestion
    verbose = verbose,
```
We should pass this in, right? Or else `verbose` won't do anything?

Reply:
I can see that it would be a lot of output if we pass in `verbose` here, but it would probably be surprising for users not to see the messages they expect with `verbose = TRUE`. What do you think?